### ISSUE TITLE
Allow customizing the display of tags in agenda

### ISSUE THREAD

**Issue by nosewings** [emacs-dashboard/emacs-dashboard#405]
It would be nice to have some control over how/whether tags are displayed in the agenda.

In my case, I just don't want tags to be displayed at all. Making this possible would be as simple as adding a new variable and then changing `dashboard-agenda-entry-format` to account for it.

**Comment by ricardoricho** [comment]
Yes @nosewings, both solutions are good. Maybe with the second provide a couple of custom functions to display or not the tags.
